SharedSource/Core/Utilities/EqualityUtility.cs

```diff
@@ -207,7 +207,7 @@ public static bool EqualsEquatable<T> (T a, T b)
     public static bool EqualsEquatable<T> (T a, object b)

```

Comment:
Potential null bug: if parameter `a` is null and `other` is not this throws an NPE, while the overload above works fine.
```csharp
      EqualityUtility.EqualsEquatable ((string) null!,  "");
      EqualityUtility.EqualsEquatable ((string) null!,  (object) "");
```
Line 2 throws a NPE while Line 1 just returns false

Candidate for bugfix ticket? @MichaelKetting

Reply:
yes, please. Not that it's particularily relevant. the EU is intended for implementing Equals and GetHashcode for an object, so the first parameter is never null.